cmd/stack/deploy.go

```diff
@@ -47,6 +50,30 @@ func Deploy(ctx context.Context) *cobra.Command {
 				return err
 			}
 
+			if okteto.IsOkteto() {
+				c, err := okteto.NewOktetoClient()
+				if err != nil {
+					return err
+				}
+				hasAccess, err := utils.HasAccessToNamespace(ctx, s.Namespace, c)
+				if err != nil {
+					return err
+				}
+				if !hasAccess {
+					log.Warning("Namespace '%s' not found", s.Namespace)
+					create, err := utils.AskYesNo(fmt.Sprintf("Do you want to create the namespace '%s'? [y/n] ", s.Namespace))
+					if err != nil {
+						return err
+					}
+					if create {
```

Comment:
is there a way to not have so much indentation. I was thinking about encapsulating alll this code in a function but there is a lot of indentation as well...

Reply:
yes, I would have a function for this. also, I would fail fast and return an error if `create` is false. it will also help with indentation